data/campaigns/tutorial03_seafaring.wmf/scripting/texts.lua

```diff
@@ -187,28 +188,37 @@ ferry_1 = {
       p(_[[Ships have the advantage that they can carry large quantities of wares and workers at a time, but unfortunately their destinations are limited to a handful of spaces suited for a port.]]) ..
       p(_[[Look at this valley here in the far south. We have found a mountain with gold down here and would like to mine ores from it as well. But the streams seperating it from our main colony are too wide to build bridges and too narrow for ships to pass them. Our roads would have to take long curves, which slows down ware transport a lot.]])
    ),
-   h = 250
+   h = 350
 }
 ferry_2 = {
    position = "topright",
+   field = shore,
    title = _"There is more to it…",
    body = (
-      h1(_"Rowboats") ..
-      p(_[[This is where ferries come in. Ferries are small rowboats that can carry only one ware at a time, and no workers (other than the one who rows it and who never leaves his boat). They can travel between any two flags located on the shore.]]) ..
-      li_image("images/wui/fieldaction/menu_build_water.png", _[[Unlike ships which travel to any port where they’re needed, ferries have fixed transport routes called waterways. A waterway is built just like a road: You click on a flag near the shore, choose ‘Build a waterway’, and select the path for the waterway just as you would do for a road.]]) ..
-      p(_[[Waterways can be built on the open sea as well as close to the shore where the water is too shallow for big ships. The only rule is that the two triangles directly adjacent to each segment of the waterway have to be water, and the entire path needs to be within your territory.]])
+      h1(_"Rowboats & Ferry Yards") ..
+      li_object("atlanteans_ferry_yard", _[[This is where ferries come in. Ferries are small rowboats that can carry only one ware at a time, and no workers (other than the one who rows it and who never leaves his boat). Ferries are built in ferry yards, one rowboat costs two logs and one piece of cloth and is quickly constructed. A ferry yard is a medium-sized building that needs to be built close to the shore.]]) ..
```

Comment:
The text is a bit long. You can split it up into multiple p() within the li_object.